xcdat/bounds.py

```diff
@@ -232,40 +248,33 @@ def get_bounds(
         return bounds
 
     def add_bounds(self, axis: CFAxisKey) -> xr.Dataset:
-        """Add bounds for an axis using its coordinate points.
+        """Add bounds for an axis using its coordinates as midpoints.
 
         This method loops over the axis's coordinate variables and attempts to
-        add bounds for each of them if they don't exist. The coordinates must
-        meet the following criteria in order to add bounds:
+        add bounds for each of them if they don't exist. Each coordinate point
+        is the midpoint between their lower and upper bounds.
+
+        To add bounds for an axis, its coordinates must be the following
+        criteria:
 
           1. The axis for the coordinates are "X", "Y", "T", or "Z"
           2. Coordinates are single dimensional, not multidimensional
           3. Coordinates are a length > 1 (not singleton)
-          4. Bounds must not already exist.
+          4. Bounds must not already exist
              * Determined by attempting to map the coordinate variable's
-             "bounds" attr (if set) to the bounds data variable of the same key.
-          5. Time axes should be composed of `cftime` objects.
```

Comment:
I'm curious why this was removed – it seems like you specifically require this in `_create_bounds`.

Reply:
With that said...I don't think we need to require datetime-like objects here (see next comment below).